### ISSUE TITLE
Corrupt forge jar after server update

### ISSUE THREAD

**Issue by ReijiGazpacho** [GTNewHorizons/GT-New-Horizons-Modpack#15370]
### Your GTNH Discord Username

reijing

### Your Pack Version

2.5.1

### Your Server

Private server (Pebblehost)

### Java Version

Java 17

### Type of Server

Vanilla Forge

### Your Actions

Tried updating server from 2.4.0 to 2.5.1. Over 200 mods and Forge itself are marked as corrupt zips and don't load, leading to 13750 missing blocks/items and preventing the world from loading.
Server was wiped and freshly installed at least 5 times, leaving only the world folder in place.

### Crash Report

[latest(4).log](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/13997483/latest.4.log)

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the crash still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this crash consistently by following the exact steps I described above.
- [ ] I have asked other people and they confirm they also crash by following the exact steps I described above.

**Comment by bombcar** [comment]
Redownload the pack

**Comment by ReijiGazpacho** [comment]
Redownloaded, FIVE TIMES. Same result every time.
Also happened after installing the 2.5.0 server:
[latest(5).log](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/13999505/latest.5.log)
Discord user thedr._ appears to be having a near-identical issue. With their permission I'm posting their log as well:
[fml-server-latest_1.log](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/13999595/fml-server-latest_1.log)

**Comment by bombcar** [comment]
Can you try java 8?

@eigenraven the only thing I can think of that could possibly cause this is lwjgl3ify somehow getting corrupt libs

**Comment by eigenraven** [comment]
Can you check how much free disk space you have by running `df -h .` in the server directory?
Java 17 is almost certainly unrelated, and the log says you're running java 8 anyway

**Comment by ReijiGazpacho** [comment]
I could've sworn I tried the java 8 version at least once prior (but forgot to keep a log), but... now it works. Thank you!

I had no problems with the java 17-20 version of 2.4.0, so I assumed Pebblehost was automatically handling the java version. I can manually set it to 17 or higher, but doing so caused the 2.5.1 J17-21 server to instantly crash. Also didn't keep a log of that, sorry.

Pebblehost doesn't make it obvious how much free disk space I can work with. I do know the server is about 5.5 GB without a backup present, and the last backup I took was about 2.3 GB.
